app/controllers/submissions_controller.rb

```diff
@@ -279,20 +279,18 @@ def score
     # To enable hints when scoring a submission, uncomment the next line:
     # send_hints(client_socket, StructuredError.where(submission: @submission))
 
-    # Submit the score
-    if @submission.users.map {|user| lti_outcome_service?(@submission.exercise, user, @submission.study_group_id) }.any?
-      url = finalize_submission_path(@submission)
-      submit_info = transmit_lti_score
-      submit_info&.fetch(:status, [])&.each do |s|
+    # The own score can only be transmitted when the lti_outcome_service is available
+    # if it is not available the user also does not send the score for the programming group members
```

Comment:
Suggestion (line could be too long and might require another break):

```rb
# A score can only be transmitted if the `lti_outcome_service` has been announced during the LTI exercise launch.
# Since the absence of the `lit_outcome_service` for the **current user** is considered a permanent error (i.e., because the deadline on Xikolo as an LMS passed), we also avoid sending a score for any (other) member of the programming group.
```

Reply:
Probably not. However, we can also remove the "old" comment (will do so).